wrangler/.env.dist

```diff
@@ -0,0 +1,13 @@
+STELLAR_NETWORK = <TESTNET|PUBLIC> # TESTNET
+HORIZON_URL = <https://horizon-testnet.stellar.org|https://horizon.stellar.org> # https://horizon-testnet.stellar.org
+TURRET_ADDRESS = <stellar-account to receive fees>
+TURRET_FUNCTION_RUNNER_URL = <URL where txFunction is hosted> # null
+WRANGLER_ACCOUNT_ID = <ID of account where CF worker is deployed>
+WRANGLER_WORKER_NAME = <Name of CF worker> # tss-wrangler
+WRANGLER_META = <ID of wrangler KV 'META'>
+WRANGLER_TX_FUNCTIONS = <ID of wrangler KV 'TX_FUNCTIONS'>
+WRANGLER_TX_FEES = <ID of wrangler KV 'TX_FEES'>
+WRANGLER_XLM_FEE_MIN = <Minumum accepted value for txFunctionFee transaction> # 1
+WRANGLER_XLM_FEE_MAX = <Maximum accepted value for txFunctionFee transaction> # 10
```

Comment:
One more field to collect before this one:

```
WRANGLER_TX_FUNCTION_FEE_DAYS_TTL = <Days before a user can reclaim fees from the turret> # 180
```

Reply:
added